packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -589,6 +590,11 @@ Please file an issue at: https://github.com/shorebirdtech/shorebird/issues/new
   if (autoUpdate != null) {
     yamlContent.writeln('auto_update: $autoUpdate');
   }
+
+  final String? shorebirdPublicKeyEnvVar = Platform.environment['SHOREBIRD_PUBLIC_KEY'];
```

Comment:
There is probably an easier way to pass the key to ios since we edit the yaml in a dart context. But in order to keep it simple and consistent to android, I thought it would make since to do the same thing here too.